packages/deployer/src/deployer.ts

```diff
@@ -69,10 +73,25 @@ export function deploySolution(
 
           common.deleteItemProps(itemBase);
 
+          // Sanitize individual parts of deployOptions because sanitizer gets rid of progress callback
+          const sanitizer = new common.Sanitizer();
+          deployOptions.title = common.sanitizeJSONAndReportChanges(
+            deployOptions.title,
+            sanitizer
+          );
+          deployOptions.snippet = common.sanitizeJSONAndReportChanges(
+            deployOptions.snippet,
+            sanitizer
+          );
+          deployOptions.description = common.sanitizeJSONAndReportChanges(
+            deployOptions.description,
+            sanitizer
+          );
+
           _deploySolutionFromTemplate(
             templateSolutionId,
-            itemBase,
-            itemData,
+            common.sanitizeJSONAndReportChanges(itemBase, sanitizer),
+            common.sanitizeJSONAndReportChanges(itemData, sanitizer),
```

Comment:
@jmhauck sorry I did not see this PR earlier - I'm just reading through this and it looks like the deployer is handling the sanitize vs delegating to the type-specific functions... is that something we could change? I ask because both Hub and StoryMaps override the base sanitizer rules to allow more properties - mainly stuff for accessibility - and running those solution templates through the base sanitizer will break them. I'll also reach out via email as well